multiversx_sdk/core/transactions_factories/token_management_transactions_factory.py

```diff
@@ -893,7 +893,7 @@ def create_transaction_for_nft_metadata_recreate(self,
     def create_transaction_for_changing_token_to_dynamic(self,
                                                          sender: IAddress,
                                                          token_identifier: str) -> Transaction:
-        parts: List[str] = [
```

Comment:
we can also add an additional check here, we cannot change nft token to dynamic

Reply:
we are not able to know token type at this point, but at least add a comment here for this

Reply:
that is right but we do not have token type as a parameter here.

Reply:
added docstring